corehq/motech/dhis2/events_helpers.py

```diff
@@ -39,14 +40,16 @@ def _get_program(config, case_trigger_info):
 def _get_org_unit(config, case_trigger_info):
     org_unit_id = None
     if config.org_unit_id:
-        org_unit_id = get_value(config.org_unit_id, case_trigger_info)
+        value_source = as_jsonobject(dict(config.org_unit_id))
+        org_unit_id = get_value(value_source, case_trigger_info)
```

Comment:
This seems to be a common pattern. Could it be refactored into a wrapper, or maybe moved into `get_value` itself?
```py
def get_config_value(config_data, case_trigger_info):
    value_source = as_jsonobject(dict(config_data))
    return get_value(value_source, case_trigger_info)
```